pages/linux/getconf.md

```diff
@@ -0,0 +1,24 @@
+# getconf
+
+> getconf - get configuration values from your Linux system.
+> 
+
+- List every configuration values available
+
+`getconf -a`
+
+- List the configuration values for a specific directory
+
+`getconf -a {{directory}}` 
+
+- Check if your linux system is a 32- or 64-bit system
+
+`getconf LONG_BIT`
+
+- Check how many processes a user can run at once
```

Comment:
```suggestion
- Check how many processes the current user can run at once:
```

- This change is partially [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout).
- Mandatory: trailing colons should be always provided.
- Optional: it's better to specify whether we are talking about the current user or not.